chalice/deploy/appgraph.py

```diff
@@ -180,7 +180,10 @@ def _create_rest_api_model(self,
         return models.RestAPI(
             resource_name='rest_api',
             swagger_doc=models.Placeholder.BUILD_STAGE,
-            endpoint_type=config.api_gateway_endpoint_type,
+            endpoint=models.Endpoint(
```

Comment:
I think it might simplify things to just normalize the `vpce_ids` to always be a list, and if a string is provided in the config to just convert it `[config.api_gateway_endpoint_vpce]`.

Reply:
Agree, I was biased by our production scenario and forgot that API Gateway in general allows you to attach multiple VPC endpoints to the private gateway.

Will change, thanks.